flashinfer/fp4_quantization.py

```diff
@@ -120,53 +136,93 @@ def _fake_fp4_quantize_sm100(
         )
 
     @register_custom_op(
-        "flashinfer::fp4_swizzle_blockscale_sm100",
-        mutates_args=("swizzled_sf",),
+        "flashinfer::block_scale_interleave_sm100",
```

Comment:
Why do we change the `swizzle` to `interleave`? I think they mean different things: swizzle is for the 128x4 layout, interleave is for the butterfly-pattern like reorder.